README.md

```diff
@@ -90,8 +90,12 @@ These packages only support .NET Framework >= 4.8, .NET Core 3.1 and .NET 6.
 
 =======
 
-## 2021-08-01 Build 2108 - August 2021 (Canary)
+## 2021-08-xx Build 2108.1 - August 2021 (Canary - Update 1)
```

Comment:
Why a separate version ? 2108 is not out yet ?

Reply:
Need a tad more time fine tuning the colours